.github/stale.yml

```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
```

Comment:
I think this needs to be larger, perhaps 180 or so?

Reply:
We could increase it to 180. The bot comments at `daysUntilStale` and then waits `daysUntilClose` until actually closing if no response is received.